astrosat_users/templates/astrosat_users/email/invitation_message.txt

```diff
@@ -1,24 +1,30 @@
 {# used for notifying a user that they have been added to a customer #}
 {# and potentially telling them to reset their password #}
 
+{% extends "astrosat_users/email/message_base.txt" %}
+
 {% load i18n %}
 
-{% autoescape off %}
+{% block body %}
+
+  {% autoescape off %}
+
+    Hi, you have been invited to {{ customer.title }}.
 
-  Hi, you have been invited to {{ customer.title }}.
+    {% if password_reset_url %}
 
-  {% if password_reset_url %}
+      Please follow the link below to create a User Account and Password.
 
-    Please follow the link below to create a User Account and Password.
+      {{ password_reset_url }}
 
-    {{ password_reset_url }}
+    {% endif %}
 
-  {% endif %}
+    {% if username %}
+      {% blocktrans %}
+        In case you forgot, your username is {{ username }}.
+      {% endblocktrans %}
+    {% endif %}
 
-  {% if username %}
-    {% blocktrans %}
-      In case you forgot, your username is {{ username }}.
-    {% endblocktrans %}
-  {% endif %}
+  {% endautoescape %}
 
-{% endautoescape %}
+{% endblock %}
```

Comment:
I don't see the `user` being used in this template, possibly in `astrosat_users/email/message_base.txt` though. Does `user` need to be added/updated in the `context`?

Reply:
No, I suppose not.